src/Catty/Setup/CatrobatSetup+Bricks.swift

```diff
@@ -115,6 +115,8 @@
             InsertItemIntoUserListBrick(),
             ReplaceItemInUserListBrick(),
             DeleteItemOfUserListBrick(),
+            WebRequestBrick(),
```

Comment:
Please add a feature toggle which makes it possible to not release this brick yet.